plugins/font/plugin.js

```diff
@@ -209,10 +163,323 @@
 			},
 
 			refresh: function() {
-				if ( !editor.activeFilter.check( style ) )
+				if ( !editor.activeFilter.check( allowedAndRequiredContent ) )
 					this.setState( CKEDITOR.TRISTATE_DISABLED );
 			}
```

Comment:
Not sure, but probably once we synchronize UI element state on command refresh, this code could be removed?

Reply:
Now it checks command state and disables it, when the command is disabled.
Unfortunately richcombo by itself determines what state should have, and only allows on disabling it in refresh method.

https://github.com/ckeditor/ckeditor4/blob/16c60f9c79c0658f37480377f9ba91d0aec5c69a/plugins/richcombo/plugin.js#L184-L187